.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+on:
+  push:
+jobs:
+  acceptance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+      - name: Setup DRP
+        run: |
+          curl -fsSL get.rebar.digital/stable | bash -s -- install --isolated --version=stable
+          ln -s $(pwd)/bin/linux/amd64/dr-provision /usr/local/bin/dr-provision
+          dr-provision --version
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: '0.13.*'
```

Comment:
Why use 0.13? this is totally EOL, and we only use 1.3.X now

Reply:
I've added tests for Terraform versions from 0.13 until 1.4 now.